doc/tests.md

```diff
@@ -0,0 +1,68 @@
+# Travis CI
+
+Every pull request runs the full suite of functional tests, and unit tests on Travis CI. We test on Chrome stable and beta, on Firefox we test ESR, latest, and beta. See our `[.travis.yml](/.travis.yml)` for the configuration, as well as [`scripts/setup_travis.sh`](scripts/setup_travis.sh) and
```

Comment:
setup_travis.sh and run_travis.sh links are broken (now that this is in the doc folder).